backend/repositories/projectRepository.js

```diff
@@ -13,20 +13,48 @@ function ProjectRepository() {
 	this.model = Project;
 };
 
-function createProject(){
-
+function createProject(projectTitle, projectDescription, participantsArr, projectSDate, projectEDate, projectVersion, projectRepository, callback){

```

Comment:
form this object in service. There are too much params to get. Use obj in this case.